src/bloaty.cc

```diff
@@ -1940,6 +1959,8 @@ USAGE: bloaty [OPTION]... FILE... [-- BASE_FILE...]
   --list-sources     Show a list of available sources and exit.
   --source-filter=PATTERN
                      Only show keys with names matching this pattern.
+  --not-source-filter=PATTERN
+                     Exclude keys with names matching this pattern.
```

Comment:
We should document what happens if a name matches both `--source-filter` and `--exclude-source-filter`.

Reply:
I wrote a test that shows what happens, basically nothing is output other than the fact that everything is omitted.

Reply:
This has been added the the help text.